plugins/version.txt

```diff
@@ -1 +1 @@
-14
\ No newline at end of file
+18
```

Comment:
Any reason for version 18 from 14?

Reply:
Hi Hrishikesh,
Modified the jpi files couple of times. So, Version has become 18

Reply:
yes. WIP.

Reply:
_can use [current_version](https://github.com/fabric8io/openshift-jenkins-s2i-config/blob/master/plugins/version.txt) + 1 to test the plugin changes_